.travis.yml

```diff
@@ -23,7 +23,7 @@ before_install:
   - cd ../..
 
 script:
-  - "mvn verify -Dei.backendInstancesListJsonContent=\"[{ 'contextPath': '', 'port': '${EI_BACKEND_PORT}', 'name': 'EI-Backend-1', 'host': 'localhost', 'https': false, 'defaultBackend': true}]\" -B"
+  - "mvn verify -P integrationTest -Dei.backendInstancesListJsonContent=\"[{ 'contextPath': '', 'port': '${EI_BACKEND_PORT}', 'name': 'EI-Backend-1', 'host': 'localhost', 'https': false, 'defaultBackend': true}]\" -B"
 
```

Comment:
Is it posible to provide all properties with -D flag now?

Reply:
Is it possible to provide it as environment Java properties as well? eg.  ei_backendInstancesListJsonContent (or with capital letters). I think it would be good to keep the varaibles in Java porpeties with only capital letters or only lower letters and not mix. E.g. ei.backend.instances.listjson.content=....  ,,, ei_backend_instances_listjson_content=....

Reply:
Yes, -D flags override the file properties, which in turn can be overwritten by environment properties.

Reply:
ok